app/src/main/kotlin/com/wire/android/ui/home/conversations/search/SearchPeopleViewModel.kt

```diff
@@ -88,7 +88,12 @@ open class SearchAllPeopleViewModel(
                 selectedContactsFlow
             ) { initialContacts, publicResult, knownResult, searchQuery, selectedContacts ->
                 SearchPeopleState(
-                    initialContacts = initialContacts,
+                    initialContacts = when(initialContacts) {
+                        is SearchResultState.Success -> {
+                            SearchResultState.Success(initialContacts.result.sortedBy { it.name.lowercase() }.toImmutableList())
+                        }
+                        else -> initialContacts
+                    },
                     searchQuery = searchQuery,
                     searchResult = persistentMapOf(
```

Comment:
What about sorting the search results, apart from the initial list? 🤔

Reply:
that's not needed for this ticket